### DISCUSSION TITLE
Nginx first setting

### DISCUSSION THREAD

**Discussion by rev0303** [NginxProxyManager/nginx-proxy-manager#3323] [General]
Hello, I would like to ask you for help with basic setting, because I am strugling here.

I am running Nginx in docker under ubuntu. Web GUI is working.

What I want to do: Have acces from internet to my server with cameras (BlueIris) - mobile application.
Below is basic explanation, as I set my network. Unfortunately, it does not work and I do not know why.

![image](https://github.com/NginxProxyManager/nginx-proxy-manager/assets/150862060/fb6147fa-b993-4c5f-8f05-6f61a6f1ebf3)

Point is, if I set port forwarding in router directly to W11 PC, it is working, but I want to go trough proxy, so I forwarded port 82 to Nginx, hoping, that rule in Nginx steams section forward to my W11 server. It does not work.

See setting below:
![image](https://github.com/NginxProxyManager/nginx-proxy-manager/assets/150862060/be84d56c-c86e-48e7-957f-44195df1e653)

In case I use browser and connect directly to my public IP with port 82, I am redirected to Nginx dashboard, so seems to me, port forwarding is working, issue is Nginx.

Do you have idea, what I doing wrong?

**Comment by olariuromeo** [comment]
I think you have to change the ip for localhost your public ip instead of 127.0.0.1 	set $server "your public ip";  or tray   create a new site that points to the application and instead of the domain set your ip and the port, it would be advisable to have an https connection

**Reply by rev0303** [reply]
Hi,

thx. I already figure it up. During container deploy, I have to add also port 82. Without that, rule in section streams does not work separately.
